app/models/event.js

```diff
@@ -57,6 +57,7 @@ export default ModelBase.extend(CustomPrimaryKeyMixin, {
   isStripeLinked  : attr('boolean'),
   canPayByCheque  : attr('boolean', { defaultValue: false }),
   canPayByBank    : attr('boolean', { defaultValue: false }),
+  canPayByOmise   : attr('boolean', { defaultValue: false }),
```

Comment:
@kushthedude Please add corresponding changes in `events.py` schema and model, otherwise it looks good to go!

Reply:
@uds5501 ,Will open the PR in Server, Thanks for reminding 😅

Reply:
@CosmicCoder96 Review Please